boards/Kconfig

```diff
@@ -1364,13 +1364,21 @@ config ARCH_BOARD_SABRE_6QUAD
 		board featuring the iMX 6Quad CPU.
 
 config ARCH_BOARD_SAMA5D2_XULT
-	bool "Atmel SAMA5D2 Xplained Ultra development board"
+bool "Atmel SAMA5D2 Xplained Ultra development board"
+depends on ARCH_CHIP_ATSAMA5D27
+        select ARCH_HAVE_LEDS
+        select ARCH_HAVE_BUTTONS
```

Comment:
Remove.  The Giant board does not have any buttons that are accessible by the user, does it?  I might be wrong.